gammapy/modeling/models/cube.py

```diff
@@ -703,6 +719,17 @@ def parameters(self):
             parameters.append(self.spatial_model.parameters)
```

Comment:
technically if you introduce the `_models` property, you could simplify this too.

Reply:
OK I will see if a larger cleanup is possible after all the covariance related PRs are in.